src/backend/distributed/commands/role.c

```diff
@@ -0,0 +1,231 @@
+#include "postgres.h"
```

Comment:
Minor: we typically have `file comments` at the start of the files with some very basic explanations of the files. Can you check some examples in the source code? And also apply it to all newly added files in this PR?